mappings/net/minecraft/client/particle/ParticleDrip.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS none/bmv
+CLASS none/bmv net/minecraft/client/particle/ParticleDrip
```

Comment:
`Droplet` is droplet from rain, `ParticleDrip` is for water/lava dripping from blocks and is used for both.